CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+
```

Comment:
The changes in this file are disallowed by `auto-changelog` (they follow the example on KeepAChangelog.com)

Reply:
Reverted in: 47b3868

However, I have thoughts about this: #60